### PR TITLE
Feature/328 added profile pic in contribution network

### PR DESCRIPTION
**Pull request by smitsekhadiaa**
## Purpose

Replace the nodes with github profile pictures of the contributors in Active Developer Collaboration network and Developer Collaboration network.

## Proposed changes

image

image

Solution to Issue #328 

#### Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/hypertrons/hypertrons-crx/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/hypertrons/hypertrons-crx)
- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,7 +26,8 @@
     "office-ui-fabric-react": "^7.183.0",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
-    "react-hot-loader": "^4.13.0"
+    "react-hot-loader": "^4.13.0",
+    "sass": "^1.52.1"
```
Comment (tyn1998) [inline package.json:30]: Hi, @smitsekhadiaa, why this package is added?

File: package.json [hunk 2]
```diff
@@ -48,8 +49,7 @@
     "html-loader": "^3.1.0",
     "html-webpack-plugin": "^5.5.0",
     "mkdirp": "^1.0.4",
-    "node-sass": "^7.0.0",
```
Comment (tyn1998) [inline package.json:51]: and this package is removed? Actually I fired an [issue](https://github.com/hypertrons/hypertrons-crx/issues/326) to plan to remove node-sass, can you provide some info?

File: package.json (hunk 1 above)
Comment (smitsekhadiaa) [reply package.json:30]: Actually I was facing some sass error while doing
 `yarn run start`
so I did `yarn add sass` and it was resolved
should I remove it , run and check weather its working properly or not?

File: package.json (hunk 2 above)
Comment (smitsekhadiaa) [reply package.json:51]: i guess i was facing the sass error due to this reason, thats why i did yarn add sass should i remove sass and try after adding node-sass?

File: package.json (hunk 1 above)
Comment (tyn1998) [reply package.json:30]: I will run `gh pr checkout 356` to pull your code into my machine and run `yarn install` then to test if it work. Please wait for a moment.

Comment (tyn1998) [reply package.json:30]: Seems everything works well as before (in fact there is no sass/scss file in Hypercrx at all but just keep it). 

I did some resesarch, `node-sass` is just a faster C++ version of sass but every time I had a  fresh `yarn install` it takes so long to build `node-sass` from scratch (I'm using ARM Mac). Glad to replace it with `sass`.

And I think `sass` is a devdependency, so please `yarn add -D` it to "devdependencies" and make an amend commit then `git push -f` to this branch(PR).

File: src/components/Graph/Graph.tsx [hunk 3]
```diff
@@ -91,6 +91,8 @@ const Graph: React.FC<GraphProps> = ({
     const generateNodes = (nodes: any[]): any => {
       const minMax = getMinMax(nodes);
       return nodes.map((n: any) => {
+      const imageURL=`image://https://avatars.githubusercontent.com/` + n.name;
+      if(n.name.includes('/')){
```
Comment (tyn1998) [inline src/components/Graph/Graph.tsx:95]: Hey @smitsekhadiaa, it is a feasible way and brings minimal change to existing code.  You can add some comment on these lines (I mean in Graph.tsx not on GitHub) to explain why there is a if-branch.

**Review by tyn1998 — CHANGES_REQUESTED** [review]

File: src/components/Graph/Graph.tsx (hunk 3 above)
Comment (smitsekhadiaa) [reply src/components/Graph/Graph.tsx:95]: yes got it @tyn1998 
i'll add the comments to the code right now.👍

File: package.json (hunk 1 above)
Comment (smitsekhadiaa) [reply package.json:30]: hey @tyn1998 I have made the required changes👍

I'll issue the PR for the corresponding changes in the readme also tonight.

Comment (tyn1998) [reply package.json:30]: > so please `yarn add -D` it to "devdependencies"

image

have to remove it first~

**Comment by smitsekhadiaa** [comment]
hey @tyn1998 
sorry for the confusion,
now kindly check, I have made the required changes

**Comment by wxharry** [comment]
Hi, I am getting an error from the format checks. Did you guys run `prettier` before merging into the master branch?

**Comment by tyn1998** [comment]
Haha, we both missed. So we need your improved workflow being merged sooooon!

**Pull request merged**